test/static.test.js

```diff
@@ -1740,7 +1740,7 @@ t.test('with fastify-compress', t => {
   })
 })
 t.test('register /static/ with schemaHide true', t => {
-  t.plan(4)
+  t.plan(3)
```

Comment:
Why? There weren't unfinished test errors before?

Reply:
Because we no longer have `GET` and `HEAD` registered separately.
The internal `HEAD` route also trigger `onRoute` hook.